model/vm.rb

```diff
@@ -121,13 +121,16 @@ def cloud_hypervisor_cpu_topology
   # Reverse look-up the vm_size instance that was used to create this VM
   # and use its name as a display name.
   def display_size
```

Comment:
It looks like `display_size` is only used in Vm serializers. We can remove it and use `vm.vm_size.name` directly.

Reply:
Good to know.  I can work on a follow-up PR that does this.